Engine/src/simulation/objects/entity/EntityInstance.java

```diff
@@ -3,18 +3,39 @@
 import simulation.properties.property.api.Property;
 
 import java.io.Serializable;
-import java.util.HashMap;
 import java.util.Map;
 
 public class EntityInstance implements Serializable {
+    private final Entity contextEntity;
     private final Map<String, Property> properties;
     private boolean isAlive;
-    public int xGridCoordinate;
-    public int yGridCoordinate;
+    private int xGridCoordinate;
+    private int yGridCoordinate;
```

Comment:
I think both should be public.
If I remember correctly, Aviad or Guy (or both) said that getter & setters without any other logic considered bad practice.

Reply:
Done